ExampleMod/Content/EmoteBubbles/MinionBossEmote.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ExampleMod.Common.Systems;
+using Terraria.GameContent.UI;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content.EmoteBubbles
+{
+	public class MinionBossEmote : ModEmoteBubble
+	{
+		public override void SetStaticDefaults() {
+			// This is the emote command name for this emote.
+			// You will want to lowercase all letters to match the vanilla command.
+			// If you don't have this set, it will be the lowercase name of the class.
+			EmoteName.SetDefault("minionboss");
```

Comment:
@JavidPack I believe we're moving away from `SetDefault` in examples, in favour of just using the translation file.

Reply:
We are...but still haven't been too consistent with doing it. At the very least we sometimes leave a comment with the old way and direct users to the localization files. I would only keep the comment in 1 example, the example that has comments and is intended to be the teaching example.